src/Core/Form/IdentifiableObject/DataHandler/OrderReturnFormDataHandler.php

```diff
@@ -0,0 +1,70 @@
+<?php
+/**
+ * 2007-2020 PrestaShop SA and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2020 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+declare(strict_types=1);
+
+namespace PrestaShop\PrestaShop\Core\Form\IdentifiableObject\DataHandler;
+
+use PrestaShop\PrestaShop\Core\CommandBus\CommandBusInterface;
+use PrestaShop\PrestaShop\Core\Domain\OrderReturn\Command\UpdateOrderReturnStateCommand;
+
+/**
+ * Saves or updates order return data submitted in form
+ */
+class OrderReturnFormDataHandler implements FormDataHandlerInterface
+{
+    /**
+     * @var CommandBusInterface
+     */
+    private $bus;
+
+    /**
+     * @param CommandBusInterface $bus
+     */
+    public function __construct(CommandBusInterface $bus)
+    {
+        $this->bus = $bus;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function update($orderReturnId, array $data): void
+    {
+        $orderReturnStateId = (int) $data['order_return_state'];
+
+        $this->bus->handle(new UpdateOrderReturnStateCommand($orderReturnId, $orderReturnStateId));
+    }
+
+    /**
+     * Order Return doesn't have a create option
+     *
+     * @param array $data
+     */
+    public function create(array $data): void
+    {
```

Comment:
It is time to decide if we trigger a warning in such empty methods (there are quite a couple of them already). @jolelievre @PierreRambaud (I thin you've suggested it in one of my PRs) Wdyt?